Gruntfile.js

```diff
@@ -0,0 +1,206 @@
+module.exports = function (grunt) {
+	'use strict';
+
+	/**
+	* LOAD TASKS
+	*/
+
+	require( 'matchdep' ).filterDev( 'grunt-*' ).forEach( grunt.loadNpmTasks );
```

Comment:
This just auto loads the tasks as opposed to doing it manually. Means you don't need to tweak this anymore going forward if you install any additional tasks later on.

Reply:
Cool!